core/resharding.go

```diff
@@ -272,3 +273,27 @@ func GetShardState(epoch *big.Int) types.ShardState {
 	}
 	return shardState
 }
+
+// GetPublicKeys returns the publickeys given epoch and shardID
+func GetPublicKeys(epoch *big.Int, shardID uint32) []*bls.PublicKey {
+	shardState := GetShardState(epoch)
+
+	// Update validator public keys
+	committee := shardState.FindCommitteeByID(shardID)
+	if committee == nil {
+		utils.Logger().Warn().Uint32("shardID", shardID).Uint64("epoch", epoch.Uint64()).Msg("Cannot find committee")
```

Comment:
warning or error?

Reply:
Warning is OK, no matter what, we cannot stuck and return.